src/components/Input/Input.css

```diff
@@ -21,3 +21,12 @@
 .input {
   position: relative;
 }
+
+@media (max-width: 768px) {
+  .input__field {
+    width: 100%;
```

Comment:
is this width necessary?

couldn't `input` set the width to 700px in desktop and `input__field` always be 100%?